app/decorators/controllers/spree/api/images_controller_decorator.rb

```diff
@@ -1,15 +1,10 @@
-# frozen_string_literal: true
-
 module Spree
   module Api
     module ImagesControllerDecorator
-      def self.prepended(base)
-        base.after_action :set_variants, only: %i[create update]
-      end
 
-      private
-
-      def set_variants
+      # just get the image and update variants and viewable types
```

Comment:
I'd remove this comment, if you want to add details about what this change does and why, please use the commit description.

Reply:
ok. removed